Gruntfile.js

```diff
@@ -82,10 +82,9 @@ function configureGrunt(grunt) {
     grunt.registerTask("jasmine-node:test-on-server", function() {
         var done = this.async();
 
-        function afterJasmineNodeCompletes(exitCode) {
-            if (exitCode !== 0) {
-                grunt.fail();
```

Comment:
`grunt.fail` is not a function. It is an object with 2 methods: `warn` and `fatal`.

http://gruntjs.com/api/grunt.fail